src/devices/adeo.ts

```diff
@@ -410,6 +410,20 @@ const definitions: Definition[] = [
             await reporting.batteryPercentageRemaining(endpoint);
         },
     },
+    {
+        zigbeeModel: ['ZB-SMART-PIRTH-V3'],
+        model: '83633205',
+        vendor: 'ADEO',
+        description: 'Smart 4 in 1 sensor',
+        fromZigbee: [fz.battery],
+        exposes: [e.battery()],
+        extend: [
+            illuminance(),
+            temperature(),
+            humidity(),
+            iasZoneAlarm({zoneType: 'generic', zoneAttributes: ['alarm_1', 'alarm_2', 'tamper', 'battery_low']}),
```

Comment:
What kind of dev
```suggestion
            iasZoneAlarm({zoneType: 'occupancy', zoneAttributes: ['alarm_1', 'alarm_2', 'tamper', 'battery_low']}),
```

This is an occupancy sensor it seems.